modules/cluster-autoscaler-cr.adoc

```diff
@@ -48,16 +48,19 @@ you assign to each pod.
 <4> Specify the maximum number of cores to deploy.
 <5> Specify the minimum amount of memory, in GiB, per node.
 <6> Specify the maximum amount of memory, in GiB, per node.
-<7> Specify the type of GPU node to deploy. Only `nvidia.com/gpu` and `amd.com/gpu`
-are valid types.
+<7> Optionally, specify the type of GPU node to deploy. Only
+`nvidia.com/gpu` and `amd.com/gpu` are valid types.
 <8> Specify the minimum number of GPUs to deploy.
 <9> Specify the maximum number of GPUs to deploy.
 <10> Specify whether the ClusterAutoscaler can remove unnecessary nodes.
-<11> Specify the period, in minutes, to wait before deleting a node after
-a node has recently been _added_.
+<11> Optionally, specify the period, in minutes, to wait before deleting a node after
```

Comment:
The values don't actually have to be in minutes, though in practice these usually would be. All of the time values in this section are just strings representing durations. So, any of the following are valid: `1s`, `1m`, `1m1s`, `1.1m`, `1h1m` etc. Anything supported by golang's [ParseDuration](https://golang.org/pkg/time/#ParseDuration).